src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs

```diff
@@ -162,4 +175,100 @@ protected override int SubscriptionsCount
             return -1;
         }
     }
+
+    private ProxyCommand<OngoingTask> CreateProxyCommandForShard()
+    {
+        var (key, taskName, type) = TryGetParameters();
+
+        var command = taskName == null ? 
+            new GetOngoingTaskInfoOperation.GetOngoingTaskInfoCommand(key, type) : 
+            new GetOngoingTaskInfoOperation.GetOngoingTaskInfoCommand(taskName, type);
+
+        return new ProxyCommand<OngoingTask>(command, RequestHandler.HttpContext.Response);
+    }
+
+    protected override async ValueTask<OngoingTaskSubscription> GetSubscriptionTaskInfoAsync(DatabaseRecord record, ClusterTopology clusterTopology,
+        SubscriptionState subscriptionState, long key)
+    {
+        DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Shiran, DevelopmentHelper.Severity.Normal, "implement for sharding - https://issues.hibernatingrhinos.com/issue/RavenDB-13113");
```

Comment:
what does it mean?

Reply:
In general I think it is fine to pass nodeTag&shardNumber to be used as a proxy..
But for the sharded handler I think it should be okay also to not pass those parameters and get only the definition from the record directly..